src/utils/hotspots/update-hotspots.js

```diff
@@ -10,28 +10,30 @@ import { getHotspotOriantaion } from './get-hotspot-orientation';
 export const updateHotspots = (container, hotspotsProps, activeImageX = 0, activeImageY = 0, movingDirection = 'x-axis') => {
   hotspotsProps.forEach((hotspotProps) => {
     const {
-      popupProps, hotspots, initialDimensions, orientation,
+      popupProps, hotspots, initialDimensions, orientation, variant
     } = hotspotProps;
 
     const hotspotOriantaion = getHotspotOriantaion(movingDirection);
     const currentImage = orientation === 'x' ? activeImageX : activeImageY;
 
-    const { anchorId, open } = popupProps;
+    const { open } = popupProps;
+    const { anchorId } = variant;
 
-    const popup = getHotspotPopupNode(anchorId, open);
-    const hotspotIcon = getHotspotIcon(anchorId);
     const hotspotsPositions = prepareHotspotsPositions(hotspots);
 
+    const currentPosition = hotspotsPositions
+      .find((hotspotPosition) => hotspotPosition.imageIndex === currentImage);
```

Comment:
It's a bit confusing, imageIndex naming refers to an index number, currentImage naming refers to the image element, so better to modify the naming to be more logical, ex. `hotspotPosition.imageIndex === currentImageIndex` if both of them are indicies